sklearn/manifold/mds.py

```diff
@@ -280,6 +320,10 @@ class MDS(BaseEstimator):
         compute metric or nonmetric SMACOF (Scaling by Majorizing a
         Complicated Function) algorithm
 
+    method: string, optional, default: smacof
```

Comment:
We may want to switch the default to svd: it solves the exact problem, and should be much faster.